lib/insights_cloud/async/vmaas_reposcan_sync_task.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'vmaas_reposcan_sync_task_base'
```

Comment:
In Katello, Dynflow actions are namespaced under `Actions::Katello::<model name>::<action name>`. So for instance, Actions::Katello::Organization::ManifestRefresh is in `app/lib/actions/katello/organization/manifest_refresh.rb`.

I see you've modeled this after two existing Dynflow actions in rh_cloud, but nothing about `InsightsCloud::Async` says that they're Dynflow actions. Also, I'm not even sure this action would fall under Async (see the other comment I am about to write.)

I feel like this would make more sense namespaced under `Actions::ForemanRhCloud::VmaasReposcanSync`. (Not `VmaasReposcanSyncTask`, because that's redundant) Thoughts?

Reply:
I think now that we don't have two events to follow, we don't really need to split to the base at all. I think we can merge the classes for now, and extract the base only if we get the second event to follow.